franklin/config/settings/base.py

```diff
@@ -200,19 +200,26 @@
             'backupCount': LOGFILE_COUNT,
             'formatter': 'simple'
         },
+        'paper_trail': {
+            'level': 'DEBUG',
+            'filters': ['require_debug_false'],
+            'class': 'logging.handlers.SysLogHandler',
+            'formatter': 'verbose',
+            'address': ('logs.papertrailapp.com', 63032)
+        }
```

Comment:
@JulianGindi - Papertrail doesn't have any security on sending logs to them. Anyone who has the above address will be able to log to us. Should this be an ENV var?